Dockerfile

```diff
@@ -25,9 +25,16 @@ RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o ma
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM alpine:latest
 WORKDIR /
+# Install kubectl and other necessary tools
+RUN apk add --no-cache curl ca-certificates && \
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && \
```

Comment:
Added kubectl to this image so that it can be used for the reboot Pods too. This way, we don't need to build a separate kubectl image and there is no need to pull another image in the cluster since the operator one has already been pulled.

Reply:
That could work, yes. I'll give it a try.

Reply:
sounds good to me, just wondering about it, not a blocker or anything. Thanks for the info!